### PR TITLE
ADD additional MongoDB performance tips

### PR DESCRIPTION
**Pull request by fgalan**

### CODE REVIEW

File: doc/manuals/admin/perf_tuning.md [hunk 1]
```diff
@@ -22,6 +22,14 @@ to use WireTiger storage engine) implements per-document lock. Thus, the lock sy
 From a performance point of view, it is recommended to use MongoDB 3.0 with WireTiger, especially
 in update-intensive scenarios.
 
+In addition, take into account the following information from official MongoDB documentation, as it may have
+impact in performance:
+
+* Check that ulimit settings in your system are ok. MongoDB provides [the following recomendations](https://docs.mongodb.org/manual/reference/ulimit)
+  As described in that document, in RHEL/CentOS you have to create a /etc/security/limits.d/99-mongodb-nproc.conf file,
+  in order to set soft/hard process limit to at least 32000 (check details in the cited document).
+* [Disable Transparent Huge Pages (THP)](https://docs.mongodb.org/manual/tutorial/transparent-huge-pages/)
```
Comment (dmoranj) [inline doc/manuals/admin/perf_tuning.md]: Queda un poco árido esto, yo metería un poco de literatura en plan: "You should also disable Transparent Huge Pages to increment the performance as explain in..."

Comment (fgalan) [inline doc/manuals/admin/perf_tuning.md]: Fixed in a27685bb5858d3c208ad372a500e17e2c13aa868

**Comment by dmoranj** [comment]
LGTM

File: doc/manuals/admin/perf_tuning.md [hunk 2]
```diff
@@ -22,6 +22,15 @@ to use WireTiger storage engine) implements per-document lock. Thus, the lock sy
 From a performance point of view, it is recommended to use MongoDB 3.0 with WireTiger, especially
 in update-intensive scenarios.
 
+In addition, take into account the following information from official MongoDB documentation, as it may have
```
Comment (kzangeli) [inline doc/manuals/admin/perf_tuning.md]: from **the** official MongoDB documentation

Comment (fgalan) [inline doc/manuals/admin/perf_tuning.md]: I'm afraid it is a bit late... the PR was already merged :(

Comment (kzangeli) [inline doc/manuals/admin/perf_tuning.md]: Yeah, I saw ...
However, never too late to fix a mistake, right? :-)

Comment (fgalan) [inline doc/manuals/admin/perf_tuning.md]: I will piggyback in a next PR.

**Pull request merged**